plugins/carla/cmake/rtmempool.cmake

```diff
@@ -0,0 +1,15 @@
+# base config
+set(carla_rtmempool_basedir carla/source/modules/rtmempool)
+set(carla_rtmempool_extra_libs ${carla_pthread_libs})
```

Comment:
`carla_pthread_libs` is not defined in this scope, making this include non-portable.

Create an INTERFACE library in a scope above that carries the required libraries and link to it in this scope.